docs/mindsdb-docs/mkdocs.yml

```diff
@@ -83,13 +83,14 @@ nav:
     - Vultr Cloud: setup/vultr.md
     - Civo Cloud: setup/civo.md
   - Connect to MindsDB:
-      - MindsDB SQL Editor: connect/mindsdb_editor.md
-      - MySQL CLI: connect/mysql-client.md
       - DBeaver: connect/dbeaver.md
+      - DBT: connect/dbt.md
```

Comment:
I've sorted by name the entries here as for me feels more natural

Reply:
Thanks